x-pack/legacy/plugins/code/public/components/admin_page/admin.tsx

```diff
@@ -123,6 +124,7 @@ class AdminPage extends React.PureComponent<Props, State> {
   };
 
   public render() {
+    track(METRIC_TYPE.LOADED, 'admin');
```

Comment:
maybe this should be called in the `componentDidMount` function rather than the `render`? as this will keep incrementing the loaded counter on every render refresh (state/prop changes etc)

Reply:
This is just a quick test for frontend. @WangQianliang will have a different PR for all the frontend metrics collecting. Let me remove this one.